docs/reference/buildx_history_trace.md

```diff
@@ -15,3 +15,36 @@ Show the OpenTelemetry trace of a build record
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+View the OpenTelemetry trace for a completed build. This command loads the
+trace into a Jaeger UI running in a local container
```

Comment:
"trace into a Jaeger UI viewer and opens it in your browser." (no local container)